release-notes/5.0/preview/api-diff/preview2/.Net/5.0-preview2_System.Text.Json.md

```diff
@@ -0,0 +1,237 @@
+# System.Text.Json
+
+``` diff
+ namespace System.Text.Json {
++    public enum DuplicatePropertyNameHandlingStrategy {
++        Error = 2,
++        Ignore = 1,
++        Replace = 0,
++    }
++    public sealed class JsonArray : JsonNode, ICollection<JsonNode>, IEnumerable, IEnumerable<JsonNode>, IList<JsonNode>, IReadOnlyCollection<JsonNode>, IReadOnlyList<JsonNode> {
```

Comment:
While this was technically in the release, we've already removed it from master... maybe we want to leave all of the JsonNode changes out?

Reply:
+1. Only L209-211 should remain in this file, assuming there's no problem with pulling the rest out.